biodatacatalyst-ui/src/main/webapp/picsureui/studyAccess/studyAccess.hbs

```diff
@@ -0,0 +1,313 @@
+    #studies-access table tbody tr td:nth-child(8),
+    #studies-access table tbody tr td:nth-child(9),
+    #studies-access table tbody tr td:nth-child(10),
+    #studies-access table tbody tr td:nth-child(11) {
+        text-align: center;
+    }
+
+    #studies-access table thead tr th:nth-child(4) {
+        text-align: left;
+    }
+
+    #studies-access table thead tr th:nth-child(3),
+    #studies-access table thead tr th:nth-child(5),
+    #studies-access table thead tr th:nth-child(6),
+    #studies-access table thead tr th:nth-child(7),
+    #studies-access table thead tr th:nth-child(8),
+    #studies-access table tbody tr td:nth-child(10) {
+        width: 1%;
+    }
+
+    #studies-access span.btn {
+        width: 90px;
+    }
+
+    .btn-blue {
+        background-color: #1a568c;
+    }
+
+    .study-lst-btn1, .study-lst-btn2 {
+        height: 2.5em;
+        background: #f4f4f6;
+        color: #000;
+    }
+    .study-lst-btn-no-hov:hover {
+        color: #000;
+        background-color: #fff;
+    }
+
+    .study-lst-btn2:hover {
+        background-color: #76a6de;
+    }
+
+</style>
+
+
+
+
+<div id="infgfx" class="clearfix">
+    <div id="infgfx-line"></div>
+    <span class="col-md-6">
+		<div class="infgfx-content">
+			<div class="gfx-title clearfix" style="margin-right:0px;width:100%">
+				<div style="width:100%;font-size:25px;font-weight:bold">Authorized Access</div>
+				<button type="button" class="btn btn-primary btn-blue">Explore Now</button>
+			</div>
+			<div class="gfx-cohorts">10 Studies</div>
+			<div class="gfx-participants">15,342 participants</div>
+			<div class="gfx-authorization">
+				<img src="studyAccess/icons/authorized-protected.svg" />
+				dbGap Approval Required
+			</div>
+			<div class="gfx-datatype">
+				<img src="studyAccess/icons/med-record.svg" style="height:100px;width:100px;margin:-20px;position:relative;top:3px"/>
+				<img src="studyAccess/icons/dna.svg" style="height:46px;width:46px;margin:0px 0px 0px -18px;transform:rotate(-45deg)" />
+				Authorized Phenotypic and Genomic Datasets
+			</div>
+			<div class="gfx-datalevel">
+				<img src="studyAccess/icons/users.svg" style="padding:0px; margin:0px 10px;"/>
+				Aggregate Counts
+			</div>
+			<div class="gfx-datalevel">
+				<img src="studyAccess/icons/user.svg" />
+				<img src="studyAccess/icons/user.svg" />
+				<img src="studyAccess/icons/user.svg" style="margin-right: 0px"/>
+				Patient Level Data
+			</div>
+			<div class="gfx-downloads">
+				<img src="studyAccess/icons/download.svg" style="padding:8px" />
+				Download Authorized Datasets
+			</div>
+			<div class="gfx-api">
+				<img src="studyAccess/icons/api-access.svg" style="padding:8px"/>
+				R and Python API Access
+			</div>
+		</div>
+	</span>
+    <span class="col-md-6">
+		<div class="infgfx-content">
+			<div class="gfx-title clearfix" style="margin-right:0px;width:100%">
+				<div style="width:100%;font-size:25px;font-weight:bold;">Open Access</div>
```

Comment:
It would be nice to extract some of the inline styles, but not a blocker

Reply:
I think that all of the styling that remains in the HTML template is one off formatting that would take up more lines of code to transform it into properly referenced elements with targeted styles